logicrunner/logicrunner_unit_test.go

```diff
@@ -59,6 +59,15 @@ import (
 
 var _ insolar.LogicRunner = &LogicRunner{}
 
+const useLeakTest = true
```

Comment:
do we still need it? I'd rather have it on everywhere where it can be on. Except parallel, where it could be optional

Reply:
I mean `func leakTestCheck`

Reply:
sec pls

Reply:
like this